instrumentation/opentelemetry-instrumentation-httpx/README.rst

```diff
@@ -0,0 +1,96 @@
+OpenTelemetry HTTPX Instrumentation
+===================================
+
+|pypi|
+
+.. |pypi| image:: https://badge.fury.io/py/opentelemetry-instrumentation-httpx.svg
+   :target: https://pypi.org/project/opentelemetry-instrumentation-httpx/
+
+This library allows tracing HTTP requests made by the
+`httpx <https://www.python-httpx.org/>`_ library.
+
+Installation
+------------
+
+::
+
+     pip install opentelemetry-instrumentation-httpx
+
+
+Usage
+-----
+
+Instrumenting all clients
+*************************
+
+When using the instrumentor, all newly created clients will automatically trace
```

Comment:
We have an issue with Flask instrumentation today because of this behaviour i.e, calling instrument only instruments clients created after the call. It would be ideal if the instrumentation was written in a way that calling it would instrument all of httpx library including already existing client instances.

Reply:
I was trying to stick mostly to the public API. Wouldn't I need to wrap something internal instead to be able to instrument previously created clients?

Reply:
Nevermind, I found something else to use.

Reply:
Instrumenting/uninstrumenting existing clients works properly now